test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/Output/ComplexTagHelpers.cs

```diff
@@ -491,7 +491,7 @@ public ComplexTagHelpers()
 #line default
 #line hidden
             Instrumentation.EndContext();
-            Instrumentation.BeginContext(1376, 14, true);
+            Instrumentation.BeginContext(1374, 14, true);
```

Comment:
Shoot you were doing such a good job making the generated HTML monotonically larger. At least you're only saving a few characters :smiling_imp: